### PR TITLE
Issue#5 cart feature frontend implementation

### PR DESCRIPTION
**Pull request by ashdaily**

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+import ShopContextProvider from "./context/ShopContext";
 
-
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+	<ShopContextProvider>
+		<App />
+	</ShopContextProvider>,
+	document.getElementById("root")
+);
```
Comment (ashdaily) [inline frontend/src/index.js:14]: Won't it be better to do this in App.js instead ?

File: frontend/src/util/Cart.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    let count = 0;
+    cart.forEach((item, index) => {
+        count = count + item.qty
+    })
+
+    localStorage.setItem("cartItems", JSON.stringify(cart));
+    localStorage.setItem("totalCartItems", count);
+
+    console.log(cart)
```
Comment (ashdaily) [inline frontend/src/util/Cart.js:39]: remove `console.log`

File: frontend/src/util/Cart.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            price: variant.price,
+            out_of_stock: 'no',
+            qty: qty
+        });
+    }
+
+    let count = 0;
+    cart.forEach((item, index) => {
+        count = count + item.qty
+    })
```
Comment (ashdaily) [inline frontend/src/util/Cart.js:34]: - move the count calculation to another function to make this function body more clean.
```
countCartItems(cartItems) => { // your implementation}
```

File: frontend/src/util/Cart.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    } else {
+        cart.push({
+            id: data.id,
+            variant_id: variant.id,
+            name: data.product_name,
+            price: variant.price,
+            out_of_stock: 'no',
+            qty: qty
+        });
+    }
```
Comment (ashdaily) [inline frontend/src/util/Cart.js:29]: move it to another non exported function, something like : 
```
const updateCart(cartItems){// copy your implementation and return latest cartItems}
```

File: frontend/src/util/Cart.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+        count = count + item.qty
+    })
+
+    localStorage.setItem("cartItems", JSON.stringify(cart));
+    localStorage.setItem("totalCartItems", count);
+
+    console.log(cart)
+}
+
+export const removeToCart = (id) => {
```
Comment (ashdaily) [inline frontend/src/util/Cart.js:42]: ~~removeToCart~~ `removeFromCart`

Comment (ashdaily) [inline frontend/src/util/Cart.js:42]: which id ? `productId` ? or `productVariantId` ? please be explicit

**Review by ashdaily — COMMENTED** [review]
@Jerry-1947  Looks good mostly, please make the small changes. 
Please remove the 3 pictures you have added.

File: frontend/src/index.js (hunk 1 above)
Comment (Jerry-1947) [reply frontend/src/index.js:14]: Need o to wrap the Context value in all the components so it is required to add Shopcontext Provider.

**Pull request merged**